model/Export/ExportForm.php

```diff
@@ -127,17 +141,38 @@ public function initElements()
 
     	$instanceElt = tao_helpers_form_FormFactory::getElement('instances', 'Checkbox');
     	$instanceElt->setDescription(__('Items'));
-    	$instanceElt->setAttribute('checkAll', true);
+
+		if(count($options) > 1){
+			$instanceElt->setAttribute('checkAll', true);
+		}
+
 		$instanceElt->setOptions(tao_helpers_Uri::encodeArray($options, tao_helpers_Uri::ENCODE_ARRAY_KEYS));
+		$instanceElt->setReadOnly(tao_helpers_Uri::encodeArray($disabledOptions, tao_helpers_Uri::ENCODE_ARRAY_KEYS));
     	foreach(array_keys($options) as $value){
-			$instanceElt->setValue($value);
+			if(!isset($disabledOptions[$value])){
+				$instanceElt->setValue($value);
+			}
 		}
 		$this->form->addElement($instanceElt);
 
 
-    	$this->form->createGroup('options', __('Export QTI 2.1 Package'), array('filename', 'instances'));
+    	$this->form->createGroup('options', $this->getFormGroupName(), array('filename', 'instances'));
```

Comment:
This should be updated depending on the QTI version.
<img width="365" alt="screen shot 2017-05-12 at 2 18 56 am" src="https://cloud.githubusercontent.com/assets/2172596/25987388/67bc79fc-36b9-11e7-8567-7e8fcd8940c1.png">